Sources/PackageModel/Manifest.swift

```diff
@@ -69,3 +69,14 @@ extension Manifest {
         return try JSON(string: PackageDescription.jsonString(package: package)).toString(prettyPrint: true)
     }
 }
+
+extension Manifest: Hashable {
```

Comment:
The name has to be unique right now because we can't handle name collisions. Manifest has access to path but it has a FIXME to remove that property. 
I added another condition that the package held by two manifest should be identical for them to be considered equal. That should be good enough for now